packages/account/src/Sections/Verification/Helpers/verification.js

```diff
@@ -14,8 +14,8 @@ export const populateVerificationStatus = account_status => {
     const has_poa = !(document && document.status === 'none');
     const has_poi = !(identity && identity.status === 'none');
     const has_submitted_poa = document_status === 'pending' && !allow_poa_resubmission;
-    const needs_poa = needs_verification.length && needs_verification.includes('document');
-    const needs_poi = needs_verification.length && needs_verification.includes('identity');
+    const needs_poa = !!needs_verification.length && needs_verification.includes('document');
```

Comment:
**NIT:** just needs_verification.length would be enough. But if u want to be explicit with boolean, can we use `Boolean()` rather than `!!` ?

Reply:
ok, will refactor to Boolean()
p.s: it is not enough, because if the length it 0, needs_poa will be 0, which is not expected, as this value is supposed to be boolean